helpdesk/update_ticket.py

```diff
@@ -302,6 +305,16 @@ def update_ticket(
         c.save()
         ticket.priority = priority
 
+    if queue != ticket.queue.id:
+        c = TicketChange(
+            followup=f,
+            field=_('Queue'),
+            old_value=ticket.queue.id,
+            new_value=queue,
+        )
+        c.save()
```

Comment:
Do you think you could take the opportunity here to refactor all the creation of TicketChange like this :

```suggestion
        c = f.ticketchange_set.create(
            field=_('Queue'),
            old_value=ticket.queue.id,
            new_value=queue,
        )
```